test/unit/gie_self_tests.cpp

```diff
@@ -819,4 +819,60 @@ TEST(gie, proj_trans_generic) {
     proj_destroy(P);
 }
 
+TEST(gie, proj_create_crs_to_crs_force_over) {
+
+    PJ_CONTEXT* ctx;
+
+    ctx = proj_context_create();
+    ASSERT_TRUE(ctx != nullptr);
+    proj_context_set_force_over(ctx, 1);
+    ASSERT_TRUE(ctx->forceOver);
+
+    auto P = proj_create_crs_to_crs(ctx, "EPSG:4326", "EPSG:3857", nullptr);
```

Comment:
Why not add the flag to the options (passed as nullptr) here?

Reply:
I've been super busy and haven't had time to revisit moving the flag around. Maybe in a few weeks I'll have some free time again.